release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1588,6 +1588,13 @@ In the following tables, features are marked with the following statuses:
 
 |====
 
+[NOTE]
+====
+Clusters that use Kuryr are not supported on {rh-openstack} 17.1. 
+
+If you have a cluster that uses Kuryr and need to run it on {rh-openstack} 17.1, apply for a support exception.
```

Comment:
If I'm still on time I would suggest rephrasing this to:

```
Clusters that use Kuryr are not supported on {rh-openstack} 17.1.

If you have a cluster that uses Kuryr and need to run it on {rh-openstack} 17.1, please contact Red Hat support for guidance and supportability details. 
```

Reply:
@sjstout ?

Reply:
@eurijon, I'm not sure I understand why you would like to direct users to support in this case. Can you provide some additional details?
I would imagine that if there is a critical business need to use Kuryr on openstack 17.1, they would call support anyway. :) 
I have concerns with driving more people to support to discuss unsupported functionality. We'd need to get an ack from support before adding something like this. 
Happy to talk further if I am missing something, which is very possible. :)

Reply:
maybe the request for support exception is automatically implied once there is no support for a version, so perhaps we can remove it from the docs?

Reply:
I reached out to PLM. They suggested that rather than directing users to support, write a KCS that details what engineering would like support to tell the customer.

Reply:
Perhaps something like the following is enough:

> Clusters that use Kuryr are not supported on {rh-openstack} 17.1. For further guidance contact Red Hat support.

Reply:
@sjstout ack. So for the release notes I believe just the following works:

> Clusters that use Kuryr are not supported on {rh-openstack} 17.1.

Reply:
I'm ok skipping the support part too, there wasn't a specific reason behind it, as @sjstout states they would call support anyway

Reply:
No, it can be closed. Thanks